core/services/collections/Collection.php

```diff
@@ -4,6 +4,7 @@
 use  EventEspresso\core\exceptions\InvalidEntityException;
 use  EventEspresso\core\exceptions\InvalidInterfaceException;
 use LimitIterator;
+use SplObjectStorage;
```

Comment:
since gutenberg branches are going to live outside of master for a bit.  I think it'd be wiser if you make these kind of updates in separate branches off master do a pull request, and merge them in.  Otherwise you increase the risk of conflicts when/if the same kind of cleanup happens in other branches.